tools/release/changelog-root-template.hbs

```diff
@@ -0,0 +1,15 @@
+{{> header}}
+
+{{#each packageGroups}}
+### {{title}}
+
+|            |                       |
+| ---------- | --------------------- |
+{{#each commits}}
+| <img src="{{typeImageUrl}}" alt="{{typeDescription}}"> | {{> commit root=@root}} |
```

Comment:
Agreed that it is pretty verbose for markdown and that it might be slightly more confusing for screen-reader users (even though we have an `alt` text).

It's a bit unfortunate as the colored labels not only look good, they also made it visually *very* clear what changed in the individual packages. Would love to explore any alternatives in the future. For now I'm happy to just use plain-text.

Reply:
Here is how it looks with just plain-text. I know we should not consider the changelog as a well-designed document, but obviously, colors can make a huge difference visually. Screenreader users are still covered by proper accessibility composition.

![image](https://user-images.githubusercontent.com/4987015/64613710-1d07f180-d3d7-11e9-880e-8b85ab555728.png)